data/result-exporter/cne-exporter-commons/src/main/java/com/powsybl/openrao/data/cneexportercommons/CneUtil.java

```diff
@@ -76,4 +92,84 @@ public static String generateUUID() {
         usedUniqueIds.add(uuidString);
         return uuidString;
     }
+
+    public static RaoParameters getRaoParametersFromProperties(Properties properties) {
+        RaoParameters raoParameters = new RaoParameters();
+        raoParameters.getObjectiveFunctionParameters().getType().relativePositiveMargins();
+        ObjectiveFunctionParameters.ObjectiveFunctionType objectiveFunctionType = getObjectiveFunctionTypeFromString(properties.getProperty(OBJECTIVE_FUNCTION_TYPE, "max-min-relative-margin-in-megawatt"));
+        raoParameters.getObjectiveFunctionParameters().setType(objectiveFunctionType);
+        boolean withLoopFlow = Boolean.parseBoolean(properties.getProperty(WITH_LOOP_FLOWS, "false"));
+        if (withLoopFlow) {
+            raoParameters.addExtension(LoopFlowParametersExtension.class, new LoopFlowParametersExtension());
+        }
+        double mnecAcceptableMarginDiminution = Double.parseDouble(properties.getProperty(MNEC_ACCEPTABLE_MARGIN_DIMINUTION, "0"));
+        if (mnecAcceptableMarginDiminution != 0) {
+            MnecParametersExtension mnecParametersExtension = new MnecParametersExtension();
+            mnecParametersExtension.setAcceptableMarginDecrease(mnecAcceptableMarginDiminution);
+            raoParameters.addExtension(MnecParametersExtension.class, mnecParametersExtension);
+        }
+        return raoParameters;
+    }
+
+    private static ObjectiveFunctionParameters.ObjectiveFunctionType getObjectiveFunctionTypeFromString(String objectiveFunctionType) {
+        if ("max-min-relative-margin-in-ampere".equals(objectiveFunctionType)) {
+            return ObjectiveFunctionParameters.ObjectiveFunctionType.MAX_MIN_RELATIVE_MARGIN_IN_AMPERE;
+        } else if ("max-min-relative-margin-in-megawatt".equals(objectiveFunctionType)) {
+            return ObjectiveFunctionParameters.ObjectiveFunctionType.MAX_MIN_RELATIVE_MARGIN_IN_MEGAWATT;
+        } else if ("max-min-margin-in-ampere".equals(objectiveFunctionType)) {
+            return ObjectiveFunctionParameters.ObjectiveFunctionType.MAX_MIN_MARGIN_IN_AMPERE;
+        } else if ("max-min-margin-in-megawatt".equals(objectiveFunctionType)) {
+            return ObjectiveFunctionParameters.ObjectiveFunctionType.MAX_MIN_MARGIN_IN_MEGAWATT;
+        } else {
+            throw new OpenRaoException("Unknown ObjectiveFunctionType %s".formatted(objectiveFunctionType));
+        }
+    }
+
+    public static CneExporterParameters getParametersFromProperties(Properties properties) {
```

Comment:
just delete the CneExporterParameters class and use the Properties instance where needed